packages/server/src/games/quiz.ts

```diff
@@ -163,8 +161,13 @@ export default class Quiz extends Room {
 
   playerGuess = (id: string, guess: string) => {
     const player = this.getPlayer(id);
+    // currentRound can be null so we check that the currentRound exists
+    if (!this.currentRound) {
+      console.log("Error: No current round")
```

Comment:
Maybe it can be nice to decide a "log strategy" if we want to keep logs

Reply:
Yes, I agree. Once we go live, I think we should definitely define Error, Warn, Info and Debug logs.